code/modules/clothing/suits/armor/plate_carrier.dm

```diff
@@ -6,15 +6,23 @@
 	color = COLOR_GRAY40
 	valid_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA, ACCESSORY_SLOT_ARMOR_C, ACCESSORY_SLOT_ARMOR_A, ACCESSORY_SLOT_ARMOR_L, ACCESSORY_SLOT_ARMOR_S, ACCESSORY_SLOT_ARMOR_M)
 	restricted_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA, ACCESSORY_SLOT_ARMOR_C, ACCESSORY_SLOT_ARMOR_A, ACCESSORY_SLOT_ARMOR_L, ACCESSORY_SLOT_ARMOR_S)
+	material = /decl/material/solid/leather
```

Comment:
feels weird to be using leather for this, maybe do cloth? 
It doesn't automatically derive armor from the material anyway.
ideally should be some sort of space kevlar, but it's all fluff differencies (and how accessible you want it to be for printing)

Reply:
w/e can be done later